### DISCUSSION TITLE
dokcer-compose modrinth example with cl5 mod in portainer wont't start

### DISCUSSION THREAD

**Discussion by Daimonion1980** [itzg/docker-minecraft-server#2782] [Q&A]
Hey guys.

I'm trying to setup a cl5 Server https://modrinth.com/modpack/create-live-5 
inside a fresh portainer lxc container on my proxmox system.
docker, including docker-compose and portainer was installed with helper script and works fine. 

After this i added a stack with the following docker-compose.yml

`
version: "3.8"

services:
  mc:
    image: itzg/minecraft-server
    tty: true
    stdin_open: true
    ports:
      - "25565:25565"
    environment:
      EULA: "TRUE"
      MOD_PLATFORM: MODRINTH
      MODRINTH_MODPACK: https://modrinth.com/modpack/create-live-5/version/1.0.4
      # or for auto-upgrading to latest
      # MODRINTH_MODPACK: https://modrinth.com/modpack/create-live-5
      # or just create-live-5 or VUmd23oG
      # and could replace version URL with
      # MODRINTH_VERSION: VUmd23oG
      # MODRINTH_VERSION: 1.0.4
      # MODRINTH_VERSION: "create live 5 Version 1.0.4"
    volumes:
      # attach the relative directory 'data' to the container's /data path
      - ./data:/data
`

the server tried to start but failed with the following log entry:

`
net.minecraftforge.fml.LoadingFailedException: Loading errors encountered: [
	Create: Dreams & Desires (create_dd) has failed to load correctly
§7java.util.ConcurrentModificationException: null
]
`

Attached the complete log file:
[_mc-java-server-cl5-mc-1_logs.txt](https://github.com/itzg/docker-minecraft-server/files/15013567/_mc-java-server-cl5-mc-1_logs.txt)

I see that the mod "Dreams & Desires" has a problem, but i don't know how to deal with it. 

Does anybody know this problem and has a tip or a solution?

**Comment by itzg** [comment]
You can exclude misbehaving mods using

https://docker-minecraft-server.readthedocs.io/en/latest/types-and-platforms/mod-platforms/modrinth-modpacks/#excluding-files

Hopefully that helps.

**Reply by Daimonion1980** [reply]
Thank you. I tried version 1.0.3 from this modpack and this worked well. Now I'm trying to find out together with the package provider what is the problem in this specific mod so, that it works even with itzg. 
Maybe disabling is a workaround here. Thanks for this tip!
